File: 2015/src/test/java/Day5Test.java
import nl.sbmf21.aoc15.Aoc;
import nl.sbmf21.aoc15.days.Day5;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day5Test {

    private final Day5 day = new Day5(new Aoc());

    @Test
    public void testPart1Output() {
        assertEquals(258, (int) day.part1());
    }

    @Test
    public void testPart2Output() {
        assertEquals(53, (int) day.part2());
    }
}

File: 2020/kotlin/src/test/kotlin/Day5Test.kt
import nl.sbmf21.aoc20.Aoc
import nl.sbmf21.aoc20.days.Day5
import org.junit.Test
import kotlin.test.assertEquals

class Day5Test {

    private var day = Day5(Aoc(), 5)

    @Test
    fun testPart1Output() = assertEquals(922, day.part1())

    @Test
    fun testPart2Output() = assertEquals(747, day.part2())
}
